notes/coredns-1.5.0.md

```diff
@@ -33,6 +33,9 @@ And a update on two important and active bugs:
 * the [*route53*](/plugins/route53) now supports wildcards.
 * the [*pprof*](/plugins/pprof) adds a `block` option that enables the block profiling.
 * the [*prometheus*](/plugins/metrics)  adds a `coredns_plugin_enabled` metric that shows which plugins are enabled.
+* the [*chaos*](/plugins/chaos) returns the maintainers when queried for "authors.bind TXT CH".
+* the `resync` in the [*kubernetes*](/plugins/kubernetes) is disabled by default and will be removed
```

Comment:
Should we declare it deprecated now? Or does saying it will be removed in the future count as being deprecated?